get-info.sh

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env bash
+
+name=$(</sys/class/dmi/id/product_name)
+
+dir="${0%/*}/$name"
+mkdir -p "$dir"
+
+echo "getting EDID for card0-eDP-1"
+cp /sys/class/drm/card0-eDP-1/edid "$dir"/card0-eDP-1.edid
+
+echo "getting dmesg"
+dmesg > "$dir"/dmesg
+
+echo "getting hwinfo"
+sudo hwinfo > "$dir"/hwinfo.txt
+
+echo "getting lspci"
+lspci -vvnn > "$dir"/lspci
+
+echo "getting lsusb"
+sudo lsusb -v > "$dir"/lsusb
+
+echo "getting Xorg.log"
+cp ~/.local/share/xorg/Xorg.0.log "$dir"/Xorg.0.log
```

Comment:
That's distribution specific. E.g. in my case with Debian/unstable I have the `Xorg.0.log` under `/var/log`